### PR TITLE
API-2010 /all swagger

### PR DESCRIPTION
**Pull request by jblefkove-va**
https://vajira.max.gov/browse/API-2010

### CODE REVIEW

**Comment by jblefkove-va** [comment]
Latest build of the swagger: [openapi.txt](https://github.com/department-of-veterans-affairs/lighthouse-facilities/files/5118523/openapi.txt)

File: facilities-api/src/main/java/gov/va/api/lighthouse/facilities/api/FacilitiesService.java [hunk 1]
```diff
@@ -107,7 +107,7 @@
 @Path("/")
 public interface FacilitiesService
     extends FacilitiesAllApi,
-        FacilitiesIdsApi,
-        FacilitiesNearbyApi,
         FacilitiesReadApi,
-        FacilitiesSearchApi {}
+        FacilitiesSearchApi,
+        FacilitiesIdsApi,
```
Comment (taylorrassmannlibertyits) [inline facilities-api/src/main/java/gov/va/api/lighthouse/facilities/api/FacilitiesService.java:112]: Just a nitpick the alphabetical order is switched here

Comment (jblefkove-va) [reply facilities-api/src/main/java/gov/va/api/lighthouse/facilities/api/FacilitiesService.java:112]: Yes, the order here impacts how they show up in the Swagger. I think this order is correct:
image

(I alphabetized them in the previous PR when the new endpoint was added without realizing this.)

Comment (taylorrassmannlibertyits) [reply facilities-api/src/main/java/gov/va/api/lighthouse/facilities/api/FacilitiesService.java:112]: OOOO gotcha! Thanks! I notice you usually alphabetize things so I wanted to make sure :)

**Pull request merged**